### PR TITLE
Create interactive list

### PR DESCRIPTION
**Pull request by Mike47ip**
The changes made to the branch are as follows:

Add a new JavaScript file and import it as a module:
it will contain methods related to the status updates (completed: true / false).
Add event listener to the checkbox (change).
Update items object's value for completed key upon user actions.
Implement a function for the "Clear all completed" button (use filter() method).
Store the updated array of items in local storage, so the user gets the correct list of values after the page reloads (which means that any changes made to the list should be preserved).

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,44 +1,28 @@
...
-       ><span class="material-symbols-outlined"> more_vert </span>
-      </li>
-     </ul>`;
-    taskcontainer.appendChild(li);
-  });
+  // Save the updated task list to local storage
+  saveTaskListToLocalStorage(incompleteTasks);
+
+  // Display the updated task list
+  displayTasks(incompleteTasks);
```
Comment (levy002) [inline src/index.js:21]: - [x] Kindly make sure to update the tasks `index` after clearing all completed tasks. Remember always task `index` should start at 1 (if the task is the fourth its index should always be 4)
```suggestion
 // Updating all task's index
  incompleteTasks.forEach((task, i) => {
    task.index = i + 1;
  });

  // Save the updated task list to local storage
  saveTaskListToLocalStorage(incompleteTasks);

  // Display the updated task list
  displayTasks(incompleteTasks);
```

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @Mike47ip    , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter error :heavy_check_mark: 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Mike47ip** [comment]
Hello bro. I do not seem to understand what changes you want me to do.We
were included to use the filter method to loop through if you can check the
requirements. Please get back to me

**Review by Owusu-Desmond — APPROVED** [review]
Hi @Mike47ip  

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**